phpstan.neon

```diff
@@ -29,3 +29,5 @@ parameters:
         - %currentWorkingDirectory%/*/Tests/*
     ignoreErrors:
         - '#Service "test_user_provider" is not registered in the container.#'
+        - '#Method Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface::dispatch\(\) invoked with 2 parameters, 1 required.#'
```

Comment:
Do we still support Symfony versions that only work with two parameters? If not I would prefer to rewrite these calls, instead of excluding it from the check.

Reply:
@danrot sadly they can not be changed  as the event subscribers are subscribed to the optional name and if we changed them they will not longer working.

But phpStan will not error when upgrading to symfony 5.0 there the second parameter is not removed and so its still possible to throw events with names:

https://github.com/symfony/symfony/blob/5.0/src/Symfony/Contracts/EventDispatcher/EventDispatcherInterface.php#L30

I think they keeped the eventName as the [workflow component](https://symfony.com/doc/4.4/workflow.html#using-events) or other dynamic event systems need this type dynamic event names.